packages/dashboards/tests/acceptance/dashboard-filter-test.js

```diff
@@ -160,7 +160,7 @@ module('Acceptance | Dashboard Filters', function(hooks) {
       {
         dimension: 'Property',
         operator: 'not equals',
-        rawValues: ['2', '3', '4']
```

Comment:
because of this `find`
https://github.com/yahoo/navi/blob/master/packages/reports/addon/components/filter-values/dimension-select.js#L78-L81
I'll make an issue